README.md

```diff
@@ -7,13 +7,30 @@ DIGITS (the **D**eep Learning **G**PU **T**raining **S**ystem) is a webapp for t
 
 # Installation
 
-If you are using Ubuntu 14.04, you can install pre-built packages for the latest major release.
-Visit [docs/UbuntuInstall.md](docs/UbuntuInstall.md) for instructions.
+| Installation method | Supported platform[s] | Available versions | Instructions |
+| --- | --- | --- | --- |
+| Deb packages | Ubuntu 14.04 | [14.04 repo](http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1404/x86_64) | [docs/UbuntuInstall.md](docs/UbuntuInstall.md) |
+| Docker | Linux | [DockerHub tags](https://hub.docker.com/r/nvidia/digits/tags/) | [nvidia-docker wiki](https://github.com/NVIDIA/nvidia-docker/wiki/DIGITS) |
+| Source | Ubuntu 14.04, 16.04 | [GitHub tags](https://github.com/NVIDIA/DIGITS/releases) | [docs/BuildDigits.md](docs/BuildDigits.md) |
 
-To build DIGITS and its requirements from source, visit [docs/BuildDigits.md](docs/BuildDigits.md) for instructions.
+# Usage
 
 Once you have installed DIGITS, visit [docs/GettingStarted.md](docs/GettingStarted.md) for an introductory walkthrough.
 
+Then, take a look at some of the other documentation at [docs/](docs/) and [examples/](examples/):
+
+* [Getting started with Torch](examples/getting-started-torch/README.md)
```

Comment:
Oops, I had been playing with it and forgot to change everything back. Good catch, thanks!